src/ast/nodes/ImportExpression.ts

```diff
@@ -296,3 +394,15 @@ const accessedImportGlobals: Record<string, string[]> = {
 	cjs: ['require'],
 	system: ['module']
 };
+
+function getDeterministicObjectDestructure(objectPattern: ObjectPattern): string[] | undefined {
+	const variables: string[] = [];
+
+	for (const property of objectPattern.properties) {
+		if (property.type === 'RestElement' || property.computed || property.key.type !== 'Identifier')
```

Comment:
I am not sure how to cover the `.computed` case, but I think it's better to have the check to be more defensive.

Reply:
Maybe something like https://rollup-46n0bvzzx-rollup-js.vercel.app/repl/?pr=4952&shareable=JTdCJTIyZXhhbXBsZSUyMiUzQW51bGwlMkMlMjJtb2R1bGVzJTIyJTNBJTVCJTdCJTIyY29kZSUyMiUzQSUyMmNvbnN0JTIwJTdCJTVCZ2xvYmFsVGhpcy51bmtub3duJTVEJTNBJTIwZm9vJTJDJTIwYmFyJTdEJTIwJTNEJTIwYXdhaXQlMjBpbXBvcnQoJy4lMkZkZXAuanMnKSUzQiU1Q25jb25zb2xlLmxvZyhmb28pJTNCJTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlMkMlMjJuYW1lJTIyJTNBJTIybWFpbi5qcyUyMiU3RCUyQyU3QiUyMmNvZGUlMjIlM0ElMjIlMkYlMkYlMjBNdXN0JTIwYmUlMjBpbmNsdWRlZCU1Q25leHBvcnQlMjBjb25zdCUyMGElMjAlM0QlMjAnc29tZXRoaW5nJyUzQiUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSUyQyUyMm5hbWUlMjIlM0ElMjJkZXAuanMlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyb3V0cHV0JTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyZXMlMjIlN0QlMkMlMjJ0cmVlc2hha2UlMjIlM0ElMjJzbWFsbGVzdCUyMiU3RCU3RA==